resources/views/navbar/items/desktop.blade.php

```diff
@@ -2,6 +2,8 @@
     'breakpoint'      => 'md',
     'navigation'      => [],
     'navigationExtra' => null,
+    'dropdownClass'   => 'flex-shrink-0 w-56',
+    'menuDropdown'    => 'menuDropdown',
```

Comment:
what's this property for? the name in itself doesn't tell me much 🤔

Reply:
@ItsANameToo it's an ID, maybe `'menuDropdownId' => 'menuDropdownId'` will be more clear?